src/assets/scss/_mixins.scss

```diff
@@ -264,5 +264,11 @@
     #campaign-highlights-beneficiaries-and-categories-desktop {
       display: block;
     }
-  }  
+  }
+}
+
+div.frc-captcha {
```

Comment:
Per discussion on previous PR, I think we should define a mixin and then import it just where needed.

Reply:
Thanks, done in ce7e438f